config-overrides.js

```diff
@@ -26,6 +27,18 @@ module.exports = function (config, env) {
     "monaco-editor": "monaco-editor/esm/vs/editor/editor.api.js"
   }
 
+  config.module.rules.unshift({
+    test: /frontend-samples.*\.(ts|tsx|scss)$/,
+    enforce: "post",
+    loader: path.resolve("./annotation-loader/index.js"),
+    options: {
+      start: /(?:START)\s*ANNOTATION\s*[0-9]+/i,
+      end: /(?:END)\s*ANNOTATION\s*[0-9]+/i,
+      identifier: /[0-9]+/i,
```

Comment:
Maybe instead of `[0-9]+` we could use `\d+\.?\d*`. That way they can be decimals as well. Helps in the case of wanting to add an annotation between 2 and 3 for example. The author would be able to use 2.1 instead of bumping everything after three.

Reply:
Might be better to allow arbitrary alphanumeric tags.  Putting the sequence into the annotations themselves will be somewhat limiting.  I'm assuming the walkthrough sequence will be driven directly from the walkthrough rather than the number of the annotation.

Reply:
For reference, the reality data sample is converted over to use this method in this PR. Maybe that would give you a better idea of how it would look.